draft-ietf-scitt-architecture.md

```diff
@@ -370,280 +365,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
-Threshold signatures may be use to protect the service key, etc.
-
-Beyond the trusted components, Transparency Services may operate additional endpoints for auditing, for instance to query the history of Signed Statements registered by a given Issuer via a certain Subject.
-Implementations of Transparency Services SHOULD avoid using the service identity and extending the Transparency Service in auditing endpoints, except if it is necessary to compute an Append-only Log consistency proofs.
-Other evidence to support the correctness and completeness of the audit response MUST be computed from the Append-only Log.
-
-### Service Identity, Remote Attestation, and Keying
-
-Every Transparency Service MUST have a public service identity, associated with public/private key pairs for signing on behalf of the service.
-In particular, this identity must be known by Verifiers when validating a Receipt.
-
-This identity MUST be stable for the lifetime of the service, so that all Receipts remain valid and consistent.
-The Transparency Service operator MAY use a distributed identifier as their public service identity if they wish to rotate their keys, if the Append-only Log algorithm they use for their Receipt supports it.
-Other types of cryptographic identities, such as parameters for non-interactive zero-knowledge proof systems, may also be used in the future.
-
-A Transparency Service MAY provide extra evidence that it is securely implemented and operated, enabling remote authentication of the hardware platforms and/or software TCB that run the Transparency Service.
-If present, this additional evidence MUST be recorded in the Append-only Log and presented on demand to Verifiers and Auditors.
-Examples for Statements that can improve trustworthy assessments of Transparency Services are RATS Conceptual Messages, such as Evidence, Endorsements, or corresponding Attestation Results (see {{-rats-arch}}).
+All Transparency Services MUST expose APIs for the registration of Signed Statements and issuance of Receipts.
 
-For example, consider a Transparency Service implemented using a set of replicas, each running within its own hardware-protected trusted execution environments (TEEs).
-Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
-This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
+Transparency Services MAY support additional APIs for auditing, for instance, to query the history of Signed Statements.
+TODO: Remove this ^. It is a layer violation, and it mixes the concept of feed, into the function of the Transparency Service.
```

Comment:
Without assuming what you're suggesting, can you elaborate: "layer violation"?

Reply:
its not clear what is "TS" and what is "SCITT API", querying history of signed statements might not be possible with "adjacent services" see https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/183